istioctl/main.go

```diff
@@ -674,22 +682,49 @@ func preprocMixerConfig(configs []crd.IstioKind) error {
 			return err
 		}
 		if config.APIVersion == "" {
-			configs[i].APIVersion = crd.IstioAPIGroupVersion.String()
+			configs[i].APIVersion = legacyIstioAPIGroupVersion.String()
 		}
 		// TODO: invokes the mixer validation webhook.
 	}
 	return nil
 }
 
+func restConfig() (config *rest.Config, err error) {
```

Comment:
nit: can you make `restConfig()` accept `kubeconfig` as an argument instead of using the global variable?

Reply:
I can but it requires me to change all upstream calls. how about i do it in another PR?